packages/editor/src/components/provider/index.js

```diff
@@ -53,7 +53,22 @@ class EditorProvider extends Component {
 
 	getBlockEditorSettings( settings, meta, onMetaChange, reusableBlocks ) {
 		return {
-			...settings,
+			...pick( settings, [
+				'alignWide',
+				'colors',
+				'disableCustomColors',
+				'fontSizes',
+				'disableCustomFontSizes',
+				'imageSizes',
+				'maxWidth',
+				'blockTypes',
```

Comment:
Case in point about https://github.com/WordPress/gutenberg/pull/14082/files#r260939419 : While the documentation has it as `blockTypes`, the actual property of the object is `allowedBlockTypes` , meaning we now don't pass through allowed block types.

I've not confirmed this as affecting the [`allowed_block_types` filter](https://developer.wordpress.org/reference/hooks/allowed_block_types/), but if it is, I think we should (a) get it fixed before 5.2 and (b) make sure we have test coverage.